lib/Fmt_ast.ml

```diff
@@ -4393,18 +4393,18 @@ and fmt_let c ~ext ~rec_flag ~bindings ~parens ~fmt_atrs ~fmt_expr ~body_loc
   in
   let fmt_binding ~first ~last binding =
     let ext = if first then ext else None in
-    let in_ indent = fmt_if_k last (fmt_in indent) in
+    let in_ =
+      if last then Some {txt= (fun indent -> fmt_in indent); loc= loc_in}
```

Comment:
Not everyone has seen a `(int -> Fmt.t) loc option` in its life.

To be clear, this is fine.